sys/include/net/nanocoap.h

```diff
@@ -861,6 +861,23 @@ static inline ssize_t coap_get_uri_query_string(coap_pkt_t *pkt, char *target,
                                (uint8_t *)target, max_len, '&');
 }
 
+/**
+ * @brief   Find a URI query option of the packet
+ *
+ * This function searches for a query option if the form "?key=value"
+ * and would in this case return the pointer to "value" when searching
+ * for "key".
```

Comment:
```suggestion
 * This function searches for a query option of the form "?key=value"
 * and would, when present, return the pointer to "value" when searching
 * for "key".
```

Also needs to mention the case where the matched key has no associated value.